meson.build

```diff
@@ -264,33 +282,53 @@ executable(
 executable(
     'unittests',
     files(
-        'framework/sysdeps/linux/malloc.cpp',
         'framework/sandstone_unittests_utils.cpp',
+        'framework/sysdeps/linux/malloc.cpp',
     ),
     unittests_sources,
-    dependencies: [
-        gtest_dep,
-        pthread_dep
-    ],
-    include_directories : [
+    dependencies: [gtest_dep, pthread_dep],
+    include_directories: [
         framework_incdir,
     ],
-    cpp_args : [
+    cpp_args: [
         '-march=haswell',
         debug_c_flags,
         default_cpp_warn,
-	'-USANDSTONE',
+        '-USANDSTONE',
     ],
-    build_by_default : false,
-    native : true,
+    build_by_default: false,
+    native: true,
 )
 
-summary({
+if get_option('fuzzing') == true
+    executable(
+        'fuzzing',
+        install: false,
+        dependencies: target_deps,
+        link_with: [
```

Comment:
Why a separate executable, instead of the regular one?

Reply:
Seemed like a good idea at the time. Using the regular one makes it simpler though, so I'll remove the separate build target.